client/src/components/styles.css.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  thumbnailImage: {
+    width: '40px',
+    height: '40px',
+    margin: '5px'
+  },
+  scaledImage: {
+    width: '578px',
+    height: '578px'
+  },
+  thumbnailBar: {
+    display: 'flex',
+    'flexDirection': 'row',
+    'justifyContent': 'center'
+  },
+  container: {
+    width: '580px',
+    height: '640px'
+  }
+};
```

Comment:
I thought that Josh specifically told us that using stylesheets wasn't the best idea for this project? When he told us to explore styling in react

Reply:
this is styling in react; it's the same as you are doing -- I just put them all into one module and exported it to separate concerns.  external style sheets are literal external .css files which I'm not doing.

Reply:
Oh I see how that would make it easier. You just import the styles into the react files instead of linking to them in html, cool!